### DISCUSSION TITLE
autorization

### DISCUSSION THREAD

**Discussion by ops244** [make-all/tuya-local#1401] [Ideas]
hi, I saw that I made a new authorization just using a QR code, can you do the same here or will it not work?
https://github.com/tuya/tuya-smart-life

**Comment by make-all** [comment]
This.is the app authorisation to the cloud . My understanding is that to use a branded app, manufacturers pay a fee to Tuya to access this authorisation system. Open source projects cannot afford that fee, so they use developer accounts for any cloud access. But none of that is relevant to local connections (except for accessing the most accessible  method of retrieving the local keys)
